### PR TITLE
RH7: Netvsc LRO/RSC support patches 

### PR DESCRIPTION
**Pull request by prashantmhetar**

### CODE REVIEW

**Comment by prashantmhetar** [comment]
Performed below testing:
1. Install, lsmod, modinfo with these changes on CentOS 7.0, 7.3 and 7.5
2. Enable and disable the SRIOV on 7.3 and 7.5
3. NTTCP transmit/receive check
3. On Windows 2019 host / Hyper-V CentOS 7.5 to ON/OFF the LRO
ethtool -K eth1 lro [on|off] -- Success (No error log)
4. On Windows 2016 host/Hyper-V CentOS 7.x to ON/OFF the LRO
ethtool -K eth1 lro [on|off] -- As expected Fails 
[root@centos70x64 src]#  ethtool -K eth0 lro off
Cannot change large-receive-offload
[root@centos70x64 src]#  ethtool -K eth0 lro on
Cannot change large-receive-offload
Could not change any device features

In progress testing for two Win2019 hosts with deployed Hyper-V VMs

**Comment by santoshx** [comment]
The base branch has been moved. Can you rebase your change and push again?

**Comment by prashantmhetar** [comment]
> The base branch has been moved. Can you rebase your change and push again?

Rebased this branch

File: hv-rhel7.x/hv/rndis_filter.c [hunk 1]
```diff
@@ -423,8 +425,8 @@ int rndis_filter_receive(struct net_device *ndev,
 
 	switch (rndis_msg->ndis_msg_type) {
 	case RNDIS_MSG_PACKET:
-		return rndis_filter_receive_data(ndev, net_dev, rndis_msg,
-						 channel, data, buflen);
+		return rndis_filter_receive_data(ndev, net_dev, channel,
+				                rndis_msg, buflen);
```
Comment (santoshx) [inline hv-rhel7.x/hv/rndis_filter.c]: You may use tabs instead of space.

**Comment by johnsongeorge-w** [comment]
Looks good.

**Pull request merged**